### DISCUSSION TITLE
Ziffer 5 wird immer falsch erkannt.

### DISCUSSION THREAD

**Discussion by oliver073** [jomjol/AI-on-the-edge-device#2266] [Q&A]
Hallo,

ich habe das Problem, dass bei mir die Ziffer 5 immer falsch erkannt wird. Meistens als 1, manchmal als 3.
Alle anderen Ziffern werden so gut wie immer korrekt erkannt.
Ich denke, die Ursache ist, dass bei meinem Zähler die Ziffer 5 im Vergleich zu den anderen Ziffern sehr schmal ist - und zwar bei allen Stellen. Man sieht das im angehängten Bild sehr gut.

Das ist bei den Stellen nach dem Komma nur ein kleines Problem, da es sich nach max. 100 Litern wieder selbst korrigiert.
Jetzt ist allerdings die Einerstelle betroffen - und ich muss bereits 1000 Liter abwarten, bis es wieder korrigiert ist. Und bald ist die Zehnerstelle bei 5 - dann wird´s lustig...

Hat jemand das gleiche Problem? Oder gar eine Lösung?
Kann man das ganze Bild für die Auswertung vielleicht in der Breite "strecken"?

Vielen Dank im Voraus!
Oliver

![Ziffer5](https://user-images.githubusercontent.com/36733364/228760726-ed2f3e8d-cc32-429c-837d-3a0cc75287bc.jpg)

**Comment by ghost** [comment]
Schau die mal die Anleitung zum Einstellen der ROI an. https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/#how-to-setup-the-digit-rois-perfectly, insbesondere "Setup using dig-class100 or dig-cont Models" 

<img width="459" alt="image" src="https://user-images.githubusercontent.com/412645/229197050-35a74ceb-dfc9-4272-8601-7c0652fc0d70.png">


Heißt bei dir oben und unten Einkürzen.

Wenn es dann noch nicht klappt bist du herzlich eingeladen mit eigenen Bildern uns zu Unterstützen. Das hilft dann auch noch mal bei der Erkennung:

https://jomjol.github.io/AI-on-the-edge-device-docs/collect-new-images/

**Reply by oliver073** [reply]
Hallo,

vielen Dank für die Antwort!
Eine genauere Einstellung hat schon eine wesentliche Verbesserung gebracht.
Manche werden zwar immer noch nicht erkannt, aber ich hab schon mal vorsorglich das Logging aktiviert.

Oliver
